modules/age.nix

```diff
@@ -103,9 +116,30 @@ in
       message = "age.sshKeyPaths must be set.";
     }];
 
+    # Create a new directory full of secrets for symlinking (this helps
+    # ensure removed secrets are actually removed, or at least become
+    # invalid symlinks).
+    system.activationScripts.agenixMountSecrets = ''
+      _count="$(basename "$(readlink /run/secrets)" || echo 0)"
+      (( ++_count ))
+      echo "[agenix] symlinking new secrets generation $_count to /run/secrets..."
+      mkdir -p "${cfg.secretsMountPoint}"
+      chmod 0750 "${cfg.secretsMountPoint}"
```

Comment:
```suggestion
      mkdir -m 0750 -p "${cfg.secretsMountPoint}"
```

Reply:
Hm, didn't we decide to do this so that shellcheck wouldn't trigger (on this part, at least)? https://github.com/ryantm/agenix/pull/27#discussion_r585941579

Reply:
:facepalm: :sweat_smile: :sweat_smile: :sweat_smile: :sweat_smile: